src/gui/sharelinkwidget.cpp

```diff
@@ -32,6 +32,11 @@
 #include <QToolButton>
 #include <QPropertyAnimation>
 
+namespace {
+    // NOTE to reviewers - I know, you will request to use QStringLiteral, but, I tried already, and, the resulting string has incorrect encoding
```

Comment:
Well, actually I'd even propose to not use QString at all here. It's better to not have types which are not trivial as global static variables. So either switch to a `const char *` here (and use `QString::fromUtf8` at point of use) or use Q_GLOBAL_STATIC.